ipamd/ipamd.go

```diff
@@ -362,45 +358,64 @@ func (c *IPAMContext) StartNodeIPPoolManager() {
 }
 
 func (c *IPAMContext) updateIPPoolIfRequired() {
+	c.retryAllocENIIP()
```

Comment:
Is this a new change? I didn't see this in previous code.

Reply:
This was the old name of some of the code in the `c.tryFreeENI()` function. Compare to the release-1.4.1 branch.